.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Jira Ticket Number
```

Comment:
Slightly tangential but do we want to also standardize branch naming? At work we do the JIRA ticket number (so the branch would be called `GB-12` for example), but up to you

Reply:
I'm of two minds of this! It's good for Jira tracking purposes, but sometimes just seeing the ticket number in a branch name doesn't immediately tell me what the branch is doing. We can discuss at our meeting.

Reply:
Also happy to do ticket name as the branch, just as long as there's some kind of 1:1 correlation with the tickets and branches I think that'll keep things more organized